packages/treat/src/createContentHash.ts

```diff
@@ -5,6 +5,9 @@ export const createContentHash = (content: any) => {
     .createHash('sha1')
     .update(JSON.stringify(content))
     .digest('base64')
+    // identifiers cannot start with double-dashes or numbers. Also if the first
+    // char is a hyphen and the 2nd isnt a letter or underscore, then replace
+    .replace(/^(((?:-{2,})?[0-9])|-[^a-z_])/i, '_$1')
+    .replace(/\+/g, '') // removes all +'s
```

Comment:
Not sure if we _should_ be so specific about this. I think a blanket remove of `/[^a-z0-9\-_]/i` would be okay, as that is spec. Oh and including ISO 10646 characters.

But that ISO stuff is abit beyond my knowledge so would appreciate some guidance here.

https://www.w3.org/TR/CSS21/syndata.html#characters

cc @mattcompiles @michaeltaranto @markdalgleish

Reply:
Now that I think about it, it's base64 coming in. So we won't have those iso characters anyway, so that non alpha regex I said above would work just fine. I'll apply that in the morning. 

Ask with respects to the original issue, we should probably run those validation rules over the classNames also. However very unlikely because of JavaScript const exports. But debug names could introduce that.